### ISSUE TITLE
Library version 1.1.1 - Lost Highlight on Items in Drop-Down Combobox

### ISSUE THREAD

**Issue by zanderson9** [vincenzopalazzo/material-ui-swing#110]
With all settings **remaining the same**  in my code, it appears that we lost "gold" highlight on Combobox items with version 1.1.1

Version 1.0.9 had the highlight correctly.

Here are the screens of the same control with 1.0.9 and 1.1.1:

**Highlight on items is present in 1.0.9**

![Combobox-highlight](https://user-images.githubusercontent.com/49419742/73998755-d5eb4180-491f-11ea-87a4-2dd25a29e935.png)

**but is gone with 1.1.1**

![Combo-no-highlight](https://user-images.githubusercontent.com/49419742/73998775-e3083080-491f-11ea-951c-07132583906e.png)

**Comment by vincenzopalazzo** [comment]
Hi @zanderson9,

Yesterday in [this commit]() I have declared the by default Label.opaque = false.

This is the [issue](https://github.com/vincenzopalazzo/material-ui-swing/issues/101) correlated and inside this I try to get a motivation for this bug and I declare the last change of the library and with the next commit, I will try to resolve the problem in short time.

Sorry.

**Comment by zanderson9** [comment]
Hi Vincent,
The JMARS application **did not** experience "[Label overwrite border when on 0 position #101](https://github.com/vincenzopalazzo/material-ui-swing/issues/101)". 
It is not to say that the issue does not exist, it does, but in JMARS we do not have such UI case where this issue manifests itself. While it is great you fixed it, **but** for JMARS losing the dropdown highlight as a side-effect of the fix for issue 101 was more harmful. 

While you are trying to implement a solution for both 101 and 110, I went ahead and updated my local version of _AbstractMaterialTheme_ class to set label opacity to "true", which returned JMARS the highlight. I then built my local jar for JMARS with that change.

I consider my change as ***temporary*** solution until you resolve it properly. But we needed to make an internal release for JMARS today, February 7, and I needed that highlight back promptly. 
So at the moment, JMARS will be using my customized version of 1.1.1, where the highlight is back.

Thank you and I will wait to hear from you about the fix.
